libamqpprox/amqpprox_maybesecuresocketadaptor.h

```diff
@@ -371,6 +457,30 @@ class MaybeSecureSocketAdaptor {
         // us to write to the socket outside the TLS tunnel.
         return d_secured && d_handshook;
     }
+
+    void onTimer(const boost::system::error_code &error)
+    {
+        if (error == boost::asio::error::operation_aborted) {
+            return;
```

Comment:
take it this isnt worth logging. e.g. timer cancelled

Reply:
yeah, it gets cancelled every time a connection hits their data rate limit, and also whenever a socket is closed.